Makefile

```
@@ -13,9 +14,10 @@ BODY_FILE  := $(if $(f),$(f),$(BODY_FILE))
 
 # === Phony Targets ===
 # Declare targets that don't represent files
-.PHONY: help update-env-secrets update-env-vars pr-create i-create \
+.PHONY: help env-update pr i-create \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)